app/scripts/templates/sign_in.mustache

```diff
@@ -45,8 +45,8 @@
            </form>
 
            <div class="links">
-             <a href="/reset_password" class="left reset-password" data-flow-event="forgot-password">{{#t}}Forgot password?{{/t}}</a>
-             <a href="/#" class="right use-different" data-flow-event="use-different-account">{{#t}}Use a different account{{/t}}</a><br/>
+             <a href="/reset_password{{queryString}}" class="left reset-password" data-flow-event="forgot-password">{{#t}}Forgot password?{{/t}}</a>
+             <a href="/{{queryString}}" class="right use-different" data-flow-event="use-different-account">{{#t}}Use a different account{{/t}}</a><br/>
```

Comment:
I don't think email=blank needs to do anything extra, "email=blank" is from a user's P.O.V. equivalent to "Use another account"; the [sign_in view will see target=blank and ignore the cached credentials](https://github.com/mozilla/fxa-content-server/blob/dc46ea016bd255f3053d82cb55fa84ba58059619/app/scripts/views/sign_in.js#L218).

As an aside, I have no problems re-visiting this code and why it's being done. I have a lot of grumbles about this section of code and cached accounts in general, it's so complex and hard to think about. First, it seems to me we should do away with storing multiple accounts locally. It's complicated. It's yucky. Nobody fully understands its behavior. Next, if we must store multiple accounts, `getChooserAccount` seems like it's misplaced - what does the model care about what account should be displayed to the user? That seems like it should be in the signin view. Next, if we must store multiple accounts and the the user chooses `Use a different account`, clearing *all* accounts seems equally erroneous. It seems we should keep all the account data and either not call `getChooseAccount` or pass in an override that returns an empty user. None of this for this PR of course.